app/assets/stylesheets/application.css.scss

```diff
@@ -8,179 +8,16 @@
  * You're free to add application-wide styles to this file and they'll appear at the top of the
  * compiled file, but it's generally better to create a new file per style scope.
  *
```

Comment:
This file will include application styles only. Madeline folder is the new system. Embedded is the for embedded/older styles. I've tried to separate out those two from each other.